examples/step-2/step-2.cc

```diff
@@ -85,13 +85,13 @@ void make_grid (Triangulation<2> &triangulation)
                outer_radius = 1.0;
   GridGenerator::hyper_shell (triangulation,
                               center, inner_radius, outer_radius,
-                              10);
+                              5 );
```

Comment:
Sorry for the late reply, but yes I can see to all these changes. I'll let
you know when I'm finished.

Conrad

On Thu, Sep 17, 2015 at 8:13 AM, Wolfgang Bangerth <notifications@github.com
> wrote:

> In examples/step-2/step-2.cc
> <https://github.com/dealii/dealii/pull/1626#discussion_r39738079>:
>
> > @@ -85,13 +85,13 @@ void make_grid (Triangulation<2> &triangulation)
> >                 outer_radius = 1.0;
> >    GridGenerator::hyper_shell (triangulation,
> >                                center, inner_radius, outer_radius,
> > -                              10);
> > +                              5 );
>
> Fair enough. Can you mention in the comment above the make_grid()
> function that it is the same mesh as in step-1, but with fewer refinement
> steps?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dealii/dealii/pull/1626/files#r39738079>.
>